src/test/java/com/google/api/codegen/testdata/php/php_library.baseline

```diff
@@ -1498,19 +1504,21 @@ class LibraryServiceGapicClient
      *     $name = '';
      *     $request = new DiscussBookRequest();
      *     $request->setName($name);
```

Comment:
Adding a blank line here would result in adding a blank line here: https://github.com/googleapis/gapic-generator/pull/2151/files?utf8=%E2%9C%93&diff=split#diff-36f32ff1b2217a381ca1eaeeea45dfe6R141 since it's between any initialization and sample call. Is that ok with you?